frontend/components/common/ProductCard.tsx

```diff
@@ -50,7 +50,7 @@ export const ProductCardImage = ({ src, alt }: ProductCardImageProps) => {
    return (
       <AspectRatio ratio={1} flexGrow={0} flexShrink={0} position="relative">
          <IfixitImage
-            sizes="30vw"
+            sizes="(max-width: 629px) 250px, (max-width: 767px) 400px, (max-width: 895px) 250px, (max-width: 1000px) 400px, 250px"
```

Comment:
This looks to me like it will use `250px` at over-`1000px` screen sizes. Is that intended?

Also, I might consider doing these the other way around: start with the mobile size, then do `min-width` for all the larger sizes. Kinda mobile-first style. Not important, so if you don't wanna, don't worry about it.

Reply:
yes. as far as I can tell on larger displays the grid that contains the products basically caps out in width so the pictures do too. But these various viewport sizes not scaling with the picture size really explains why our responsive image sizing isn't great rn

Reply:
I mean that, if I'm reading these numbers right, we have a 400px image available, but we only use it for screens between 630px and 767px or between 896px and 1000px.

Reply:
yep thats the intended sizing. feel free to verify for yourself if youd like i was doing my best to handcode this so i could be wrong at what viewport size should be what picture size